.github/workflows/build.yml

```diff
@@ -40,6 +40,11 @@ jobs:
         with:
           name: artifacts
           path: target/*.jar
+      - name: Check dependencies for releases
+        if: startsWith(github.ref, 'refs/tags/')
```

Comment:
Don't run on tagged commits, but rather on release-branches. Failing here might be too late, as in some repos a lot of automation happens when pushing tags.

Reply:
Changed in https://github.com/cryptomator/cryptofs/pull/192/commits/60b98e43a14a8b3d7e2bb835307cbef05c4ecf4b